umn/source/faqs/how_to_access_kibana_from_elb.rst

```diff
@@ -0,0 +1,163 @@
+.. _css_02_0201:
+
+How To Access Kibana From ELB
+=============================
+
+ELB Configuration Script
```

Comment:
I think the very first subchapter should be the overview about the scenario. Why and when customer might need to use this solution...